packages/sanity/src/core/form/inputs/InvalidValueInput/converters.ts

```diff
@@ -56,6 +57,32 @@ export const converters: {[fromType: string]: {[toType: string]: ValueConverter}
         }
       },
     },
+    portableText: {
+      test: TRUE,
+      convert: (value: string) => [
+        {
+          _type: 'block',
```

Comment:
Users can alias a block type to a different name, so `block` is not safe to use - we need to find and pass through the expected type and pick the `name` property from the schema type in question.